initializr-service/src/main/resources/application.yml

```diff
@@ -477,6 +477,16 @@ initializr:
           groupId: io.zipkin.java
           artifactId: zipkin-server
           starter: false
+    - name: Cloud Task
+      bom: cloud-bom
+      versionRange: 1.3.6.RELEASE
+      content:
+        - name: Tasks
+          id: tasks
+          description: task result tracking along with integration with batch and streams
+          groupId: org.springframework.cloud
+          artifactId: spring-cloud-task-starter
+          version: 1.0.1.RELEASE
```

Comment:
Out of curiosity, Will this project be integrated with the spring cloud release train?